### PR TITLE
Event consumers for sending emails (relates #2).

### PR DESCRIPTION
**Pull request by mitjok**

### CODE REVIEW

File: app/consumers/peatio/market/btcusd_order_created_consumer.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Peatio
+  module Market
+    class BtcusdOrderCreatedConsumer
```
Comment (yssavchuk) [inline app/consumers/peatio/market/btcusd_order_created_consumer.rb]: Do we really need to use single consumer per market ?

**Comment by ysv** [comment]
@mitjok finish this pr please

File: app/consumers/peatio/model/deposit_updated_consumer.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Peatio
+  module Model
+    class DepositUpdatedConsumer
+      def call(event)
+        record = event.dig(:record)
+        transaction = OpenStruct.new(
+          email: record[:email],
```
Comment (martianatwork) [inline app/consumers/peatio/model/deposit_updated_consumer.rb]: How are we getting email from the event?

Peatio dosent send email in the event.

**Comment by ruiandrada** [comment]
@ysv @mitjok 

Hello, does this pull request continue with active development?
Is it still a good idea to continue this development?
Can I try to complete this request?

**Pull request closed without merging**